src/plugins/esm.ts

```diff
@@ -20,8 +20,7 @@ const __dirname = __cjs_path__.dirname(__filename);
 const require = __cjs_mod__.createRequire(import.meta.url);
 `
 
-const ESMStaticImportRe =
-  /(?<=\s|^|;)import\s*([\s"']*(?<imports>[\p{L}\p{M}\w\t\n\r $*,/{}@.]+)from\s*)?["']\s*(?<specifier>(?<="\s*)[^"]*[^\s"](?=\s*")|(?<='\s*)[^']*[^\s'](?=\s*'))\s*["'][\s;]*/gmu
+const ESMStaticImportRe = /import\s+.+?\s+from\s+['"].+?['"];(?![\s\S]*import\s+.+?\s+from\s+['"].+?['"];)/gmu
```

Comment:
Basically this RegEx is based on negative lookahead feature.

This pattern breaks down as follows:
- `import\s+`: Matches the word "import" followed by one or more whitespace characters.
- `.+?\s+`: Matches one or more characters (non-greedy) followed by whitespace, representing the imported entity.
- `from\s+['"]`: Matches the word "from" followed by whitespace and then either a single or double quote.
- `.+?['"];`: Matches one or more characters (non-greedy) followed by a single or double quote and a semicolon, representing the module path.
- `(?![\s\S]*import\s+.+?\s+from\s+['"].+?['"];)`: A negative lookahead that ensures there is no other import statement following this pattern in the rest of the text.

Reply:
@alexey-detr  Thank you for your contribution. This change does not work for many static import cases. 
I've learned about the JSDoc issue, but I haven't found a better way without big changes.

Reply:
@alex8088 Thank you for looking into this. Yeah, this is also not a best workaround, it seems it'll be necessary to utilize some JS/TS parser in order to make it work properly.

I found a workaround for my case where I just use vite replace plugin and drop problematic places in JSDoc. Quite a hacky way, but fine for me 🙂

Feel free to close this PR since it doesn't work for many static import cases.